src/Exceptions/JobNotSupportedException.php

```diff
@@ -18,4 +19,30 @@ public function __construct(DriverInterface $driver, JobInterface $job, int $cod
         $message = "$driverClass doesn't support jobs of $jobClass.";
         parent::__construct($message, $code, $previous);
     }
+
+
+    /**
+     * @inheritDoc
+     */
+    public function getName(): string
+    {
+        return 'Job is not supported by current queue driver';
```

Comment:
Can we replace "current queue driver" with actual driver name? Can we replace "Job" with something more concrete?

Reply:
Yes, will describe this in `getSolution`.